C/CompilerSupportHeaders/CompilerSupportHeaders@v0.5/build_tarballs.jl

```diff
@@ -0,0 +1,5 @@
+include("../common.jl")
+
+build_csh(ARGS, v"1.0.5"; preferred_gcc_version=v"12", julia_compat="1.9")
```

Comment:
Did you swap the `build_csh` calls between 0.5 and 1.0 here?

Reply:
Yes thanks, I deleted those anyway to register 1.1 for now